unic/ucd/bidi/src/bidi_class.rs

```diff
@@ -109,7 +109,9 @@ pub enum BidiClassCategory {
 impl BidiClass {
     /// Find the character *Bidi_Class* property value.
     pub fn of(ch: char) -> BidiClass {
-        bsearch_range_value_table(ch, BIDI_CLASS_TABLE)
+        // UCD/extracted/DerivedBidiClass.txt: "All code points not explicitly listed
+        // for Bidi_Class have the value Left_To_Right (L)."
+        *BIDI_CLASS_TABLE.find_or(ch, &L)
```

Comment:
Right. Let's keep it as it is for now, then, and we can clean up after adding a couple more of properties.